lua/nvim-tree/view.lua

```diff
@@ -74,16 +74,24 @@ local function matches_bufnr(bufnr)
   return false
 end
 
-local function wipe_rogue_buffer()
+function M.wipe_rogue_buffer()
+  local was_wiped = false
   for _, bufnr in ipairs(vim.api.nvim_list_bufs()) do
     if not matches_bufnr(bufnr) and utils.is_nvim_tree_buf(bufnr) then
+
+      -- if restored from session, values might be incorrect
+      for option, value in pairs(BUFFER_OPTIONS) do
+        vim.bo[bufnr][option] = value
+      end
```

Comment:
this is outside the scope of this function though.
Also if the buffer is deleted, i'm not sure what's the point of resetting its options.

Reply:
I was having to do this due to a misconfiguration in my setup. I'm having another issue now. Going to close this PR for now until I fix it.